frontend/ios/fastlane/Fastfile

```diff
@@ -92,6 +92,8 @@ platform :ios do
        output_directory: "#{ENV['HOME']}",
        output_name: "#{build_config_name}.ipa",
        export_method: "app-store",
+       # pass base64 encoded env variable "environment=production" directly, because shell result does not work for gym as it does for gradle (android/Fastfile)
+       xcargs: "DART_DEFINES='ZW52aXJvbm1lbnQ9cHJvZHVjdGlvbg=='"
```

Comment:
can you please put this base64 string into a named variable so we know what is happening here.